app/assets/stylesheets/bubble.scss

```diff
@@ -0,0 +1,50 @@
+$white: #ffffff;
+$oil: #333333;
+
+.bubble {
+  position: relative;
+  width: auto;
+  min-height: 200px;
+  padding: 5px;
+  background: $white;
+  -webkit-border-radius: 18px;
+  -moz-border-radius: 18px;
+  border-radius: 18px;
+  border: #FFFFFF solid 2px;
+  -moz-box-shadow:  -2px 2px 2px 0px rgba(67, 65, 65, 0.99);
+  -webkit-box-shadow:  -2px 2px 2px 0px rgba(67, 65, 65, 0.99);
```

Comment:
Colon after property should be followed by one space<br>`0px` should be written without units as `0`